### DISCUSSION TITLE
Version 2.3.2

### DISCUSSION THREAD

**Discussion by BourgeoisLab** [kleiner-brauhelfer/kleiner-brauhelfer-2#232] [General]
- Neu: Neue Webseite mit Dokumentation http://kleiner-brauhelfer.de/ (noch in Arbeit) (vielen Dank Pitastic und alle, die daran gearbeitet haben)
- Neu: Reiter "Rohstoffe", "Brauübersicht", "Ausrüstung" und "Datenbank" neu als Fenster (vielen Dank psytrap)
- Neu: Obergrenze der Zutaten auf 9999 erhöht
- Fix: Kombination Spunden und Zuckerlösung
- Fix: Leerzeichen und Zeilenumbrüche in den Bemerkungsfelder

<hr /><em>This discussion was created from the release <a href='https://github.com/kleiner-brauhelfer/kleiner-brauhelfer-2/releases/tag/v2.3.2'>Version 2.3.2</a>.</em>

**Comment by Rasinari** [comment]
Liebe Alle,
ich bin derzeit zu sehr eingespannt, um mich um die Doku zu kümmern. Die Vorlage ist super geworden und gefällt mir sehr. Sehr aufgeräumt. So soll es sein. Ich kann mal im Februar versuchen, die bisherigen Kapitel zu redigieren und aufzubereiten. Vor April habe ich aber keine Zeit, mich intensiver um die Doku zu kümmern.

Grüße
R.

P.S.: Bitte die Macianer nicht vergessen. Kompiliert jemand?

**Comment by nguba** [comment]
Many thanks for the great program.  Something I would suggest is some structured method to record yeast propagation steps and resulting cell count.  I am currently using the comment sections for this, so it’s not urgent.  Just a suggestion to make it even better.

A small bug bear  of mine is that all colours are single digits only.  Again, not that important but it would be better to see these at a precision of one decimal point, don’t you think?

Gut Sud!

**Comment by Rasinari** [comment]
I also use the comment section. All good.

The topic has been discussed many times before. Currently there is no increased need to change anything in propagation or fermentation management
The reason is simple: Propagation is not (yet) part of the actual brewing process. At best, a so-called starter is a small brew in its own right.

Can you clarify the point about the colors? It's not clear what you mean.

**Reply by nguba** [reply]
About colours.  Some malts have fractional values.  A pilsner malt that comes as 3.5 EBC has to be entered as either 3EBC or 4EBC.  The resulting estimate is also just a single digit.  say 7 instead of 6.8.  It would be useful to have a little more precision here.

**Comment by Rasinari** [comment]
Thank you for explanation. d'accord - I agree. A change would be useful.

**Comment by LondonRain** [comment]
I disagree. No one can notice the difference in half a EBC. And no maltster can guarantee the malt colour down do half an EBC. It will only clutter the UI more with no additional benefit.

**Comment by BourgeoisLab** [comment]
Related to issue #218.
Maybe it's enough to add one digit in the Rohstofftabelle only.

**Comment by Rasinari** [comment]
Ich mache mal auf Deutsch weiter. Halbe EBC sind gang und gäbe. Vor mir steht ein Sack Weyermann Extra Pale. EBC 2,5 - 3,5. Noch Fragen?

Wenn ja: Bei extrem hellen Bieren macht das gerade bei Hobbybrauern Sinn, die nicht über die Techniken der Industrie verfügen. Das gilt auch für extrem helle Caramalze wie sie in Belgien erhältlich sind, etwa das Cara Clair: 4,5 EBC.

Brauer aus der Oxidationsfraktion dürften halbe EBC sehr begrüßen

**Reply by BourgeoisLab** [reply]
> Halbe EBC sind gang und gäbe. Vor mir steht ein Sack Weyermann Extra Pale. EBC 2,5 - 3,5. Noch Fragen?

Hier sagst du es ist gang und gäbe und in #218 Zitat "Kommawerte bei EBC mögen da sicher eine Abhilfe sein. Ich gebe aber zu bedenken, dass das eine sehr ungewöhnlicher Lösungsweg wäre . Zumindest habe ich noch nie einen nicht-ganzzahligen EBC-Wert gesehen. "

Ich sehe es eher wie LondonRain. Das ist eine irreführende Genauigkeit, weil die Berechnung weitere Faktoren enthält (z.B. Stammwürze & Kochzeit), die den Wert viel mehr beeinflussen. Den Unterschied wird man eh nicht sehen.

**Reply by LondonRain** [reply]
> Ich sehe es eher wie LondonRain. Das ist eine irreführende Genauigkeit, weil die Berechnung weitere Faktoren enthält (z.B. Stammwürze & Kochzeit), die den Wert viel mehr beeinflussen. Den Unterschied wird man eh nicht sehen.

Danke. Exakt. In dem obigen Beispiel tippt man halt einfach 3 ein und feddich.

**Reply by Rasinari** [reply]
> > Halbe EBC sind gang und gäbe. Vor mir steht ein Sack Weyermann Extra Pale. EBC 2,5 - 3,5. Noch Fragen?
> 
> Hier sagst du es ist gang und gäbe und in #218 Zitat "Kommawerte bei EBC mögen da sicher eine Abhilfe sein. Ich gebe aber zu bedenken, dass das eine sehr ungewöhnlicher Lösungsweg wäre . Zumindest habe ich noch nie einen nicht-ganzzahligen EBC-Wert gesehen. "
> 
> Ich sehe es eher wie LondonRain. Das ist eine irreführende Genauigkeit, weil die Berechnung weitere Faktoren enthält (z.B. Stammwürze & Kochzeit), die den Wert viel mehr beeinflussen. Den Unterschied wird man eh nicht sehen.

Keine Regel ohne Ausnahme. Ich habe aktuell gerade zwei Malze mit ungeraden EBC auf Lager. Den Unterschied siehst Du, weil sich die Farboxidation weniger hinten heraus addiert.
Bei sehr hellen Bieren wird der Range schmaler. Ich denke, dass Weyermann und ein paar wenig andere in diesem Fall nicht grundlos halbe EBC anführen. 
https://www.weyermann.de/product/weyermann-premium-pilsner-malz-extra-hell/
https://twojbrowar.pl/en/beer-ingredients/malts/halcyon-pale-ale-malt-1kg-thomas-fawcett
https://twojbrowar.pl/en/beer-ingredients/malts/maris-otter-malt-1kg-thomas-fawcett

Gute Braushops wie Twoj Brovar mitteln inzwischen.

**Comment by nguba** [comment]
Manche programme haben das schon.   Siehe das Kaptitel [Teilschuettungen](http://www.brewrecipedeveloper.de/misc/brdh/Brew%20Recipe%20Developer%20Dokumentation%20DE.html?Teilschttungen.html) im Brew Recipe Developer.  Brewersfriend u.a. Arbeiten auch mit einer Nachkommastelle.

Greetings from Manchester.

**Comment by Rasinari** [comment]
Wenn es, wie Frédéric, im weiteren Berechnungsverlauf unerheblich ist, macht es nicht viel Sinn. Wenn ich dadurch aber bei einem hellen Bier 1 EBC gewinne, macht es durchaus Sinn.

Das Beispiel von nguba überzeugt mich: Es macht Sinn.

Ich gebe ein Beispiel: Ich muss ein Altbier in den Grenzen des BJCP brauen. Ich bin an der maximalen Obergrenze der Stammwürze und bekomme die 40EBC nur hin, wenn ich ein helleres Basismalz nehme und damit einen zu hohen Alkoholwert vermeide. (Den Fall hatte ich vor einem halben Jahr tatsächlich. Letztlich flog ein Teil Münchener II raus, das ich aber gerne behalten hätte.

**Comment by LondonRain** [comment]
Die Hypothese dass sich damit vermeintlich genauere Vorhersagen zur Bierfarbe treffen lassen ist einfach schon grundfalsch. Nochmal: Kein Mälzer kann garantieren, dass ein Malz auf ein halbes EBC genau ist. Für mich ist das Thema nach wie vor kompletter Unsinn und gaukelt nur nicht existierende Pseudogenauigkeit bei Gui-Zumüllung vor.

**Reply by Rasinari** [reply]
Das kann man so nicht gelten lassen. Das ist doch ein Streit um des Kaisers Bart. Sie ist nicht grundfalsch, noch grundrichtig. Sie zeigt nur auf, dass man sich dem Thema gewissenhaft nähert.

Lies Dir meinen Edit durch. Ich meine es gibt Beispiele, wo es seine Berechtigung hat. Die Frage ist: wie oft wird so etwas benötigt? Wenn ich mir die Rezeptfragen in Hobbybrauer.de anschaue, ziemlich oft sogar.

Nicht falsch verstehen: Ich bin kein Fan von Einzelinteressen, die nicht durch das Gesamtinteresse gedeckt sind. Bei dem Thema Digits geht es aber um zwei Denkschulen, so ist mein Eindruck.

Im übrigen mittle ich in den Rohstoffen alle meine EBC-Einträge. Da kommen sehr oft halbe Werte vor.

Vorschlag: einen halben Wert 0,5 zulassen.

**Comment by BourgeoisLab** [comment]
Also: man wird in Zukunft eine Dezimalstelle angeben können. Diese wird aber nur im Rohstoffdialog zu sehen sein. Ausserhalb wird der EBC in einer Ganzzahl angezeigt (Rezept/Spickzettel/usw.).

**Reply by Rasinari** [reply]
Da kann ich nur sagen thump up! Sehr gute Lösung.

**Comment by nguba** [comment]
Also, es ist euere Entscheidung.  Ich wuerde information nicht loeschen.  Alle malzanalysen sind Standesgemaes auf einer Nachkommastelle angegeben.  Warum sollte man das loeschen?  Macht keinen Sinn.  Auch bei der Endziffer is die Kommastelle wichtig.  

Fuer Exil-Bayern die seit 30 Jahren im Ausland leben, is das Brauen heller Biere schon Ueberlebenswichtig.

Da ist auch die Kommastelle wichtig.   Fuer die die es nicht wollen,  koennte man die Kommastelle Einstellungsbedingt weglassen.

Es ist aber wirklich kein drama.  Die die meinen das man es genau wissen muss koennen auch mit mehreren Programmen arbeiten.  

Schad waers halt.

**Reply by Rasinari** [reply]
Also ich braue streng nach BJCP. Da will ich gerade Werte im Ergebnis haben. Der Rechenweg benötigt aber Digits. Es gibt sonst zu viele Rundungsabweichungen. In der Rohstoffliste ist die Ziffer nach der Kommastelle wichtig. An anderer Stelle würde sie mich sehr stören.
